controllers/newContent.controller.js

```diff
@@ -0,0 +1,131 @@
+const Content = require("../Models/contentSchema");
+
+// Create and Save a new content
+exports.create = (req, res) => {
+  // Create content
+
+  const content = new Content(req.body);
+  // const content = new Content({
+  //   type: req.body.type,
+  //   contentUrl: req.body.contentUrl,
+  //   title: req.body.title || "Untitled",
+  //   description: req.body.description
+  // });
+  console.log(content);
+
+  // Save content in the database
+  content
+    .save()
+    .then(data => {
+      res.send(data);
+    })
+    .catch(err => {
+      res.status(500).send({
+        message:
+          err.message || "Some error occurred while creating the Content piece."
+      });
+    });
+};
+
+///////////////////////
+// Retrieve and return all content from the database.
+exports.findAll = (req, res) => {
+  Content.find()
+    .then(contents => {
+      res.send(contents);
+    })
+    .catch(err => {
+      res.status(500).send({
+        message: err.message || "Some error occurred while retrieving content."
+      });
+    });
+};
+
+//////////////////////
+// Find a single content with a contentId
+exports.findOne = (req, res) => {
```

Comment:
Don't export single functions. Export them together in a module pattern.

`module.exports = {
    fn1,
    fn2
}`

Reply:
Resolved.